jack-core/src/com/rapleaf/jack/transaction/DbMetrics.java

```diff
@@ -0,0 +1,23 @@
+package com.rapleaf.jack.transaction;
+
+public interface DbMetrics {
+  long getTotalConnections();
+
+  long getOpenedConnectionsNumber();
+
+  double getMaxConnectionsProportion();
+
+  long getMaxConnectionWaitingTime();
+
+  double getAverageConnectionWaitTime();
+
+  double getAverageConnectionExecutionTime();
+
+  double getMinConnectionsProportion();
+
+  double getAverageIdleConnections();
+
+  double getAverageActiveConnections();
+
+  String log();
```

Comment:
Since this method returns a string instead of doing the logging itself, probably better to rename it to `summary`, `getSummary`, or something else.